packages/views/screens/managers/editVote/index.js

```diff
@@ -44,7 +44,7 @@ const AddVote = ({
 }) => {
   const dispatch = useDispatch();
   const { account, network, voting } = useSelector(state => state);
-  const host = useSelector(state => state.account.info.LSK.summary.address);
+  const host = useSelector(state => state.wallet.info.LSK.summary.address);
```

Comment:
we need to avoid hardcoded token name on the redux `state.wallet.info.LSK.summary.address`